### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pavelzw** [comment]
@conda-forge/logfire this is ready for review

**Review by samuelcolvin — APPROVED** [review]
Lgtm

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,12 +41,15 @@ test:
...
+  doc_url: https://docs.pydantic.dev/logfire
+  summary: The best Python observability tool! 🪵🔥
   license: MIT
   license_file: LICENSE
 
 extra:
   recipe-maintainers:
     - Kludex
     - samuelcolvin
+    - pavelzw
```
Comment (samuelcolvin) [inline recipe/meta.yaml:55]: Humm, are you sure you need to be a maintainer?

I'm not sure about that since you don't otherwise maintain Logfire.

Comment (pavelzw) [reply recipe/meta.yaml]: i don't _need_ to be maintainer, was just offering to also help with failing CI as i did with the pydantic-feedstock.
removed myself in [`0f6cfb2` (#15)](https://github.com/conda-forge/logfire-feedstock/pull/15/commits/0f6cfb2be4780760ad9b5088820c6a595bea9e54)

**Review by Kludex — APPROVED** [review]

**Pull request merged**